tests/python/pytorch/dataloading2/test_itemset.py

```diff
@@ -0,0 +1,115 @@
+import dgl
+import torch
+from torch.testing import assert_close
+
+
+def test_ItemSet():
+    # Node or edge IDs.
```

Comment:
In principle, we should break each test case such as:

test_ItemSet_Node_Ids():

test_ItemSet_Graph_Ids():

test_ItemSet_Node_Pairs():

e.t.c.

Reply:
changed as suggested.